docs/content/installation/metrics/metrics-configuration.md

```diff
@@ -51,33 +57,55 @@ The specific custom configuration settings available are as follows:
 
 | Name | Default | Required | Description |
 |------|---------|----------|-------------|
-| `prometheus_custom_configmap` | crunchy-prometheus |  | The name of a ConfigMap containing a custom `prometheus.yml` configuration file. |
+| `alertmanager_custom_config` | alertmanager-config |  | The name of a ConfigMap containing a custom `alertmanager.yml` configuration file. |
+| `alertmanager_custom_rules_config` | alertmanager-rules-config |  | The name of a ConfigMap container custom alerting rules for Prometheus. |
 | `grafana_datasources_custom_config` | grafana-datasources |  | The name of a ConfigMap containing custom Grafana datasources. |
 | `grafana_dashboards_custom_config` | grafana-dashboards |   | The name of a ConfigMap containing custom Grafana dashboards. |
+| `prometheus_custom_configmap` | crunchy-prometheus |  | The name of a ConfigMap containing a custom `prometheus.yml` configuration file. |
 
 _Please note that when using custom ConfigMaps per the above configuration settings, any defaults
 for the specific configuration being customized are no longer applied._
 
+## Using Alertmanager
+
+The Alertmanager deployment requires a custom configuration file to configure reciever
+integrations that are supported by Prometheus Alertmanager. The installer will create
+a configmap containing an example Alertmanager configuration file created by
+the pgMonitor project, this file can be found in the [pgMonitor repo](https://github.com/CrunchyData/pgmonitor/blob/master/prometheus/crunchy-alertmanager.yml).
```

Comment:
Should we say `repository` instead of `repo`? (I can't tell from a quick search of the docs; there's a lot of `backrest-repo`.)